Shared/ThresholdKeyView.swift

```diff
@@ -65,7 +66,10 @@ struct ThresholdKeyView: View {
                                 }
                                 
                                 // TODO: Add proper messages, 1/2 and 2/2 (old accounts, new accounts)
-                                alertContent = "\(totalShares) shares created"
+                                alertContent = "\(totalShares - required_share)/\(totalShares) shares created."
```

Comment:
This message is incorrect.

If you have two shares and one is required, you do not have "1/2 shares created".

Reply:
editied contition and message